web/src/main/java/org/springframework/security/web/access/ExceptionTranslationFilter.java

```diff
@@ -239,6 +243,17 @@ public void setMessageSource(MessageSource messageSource) {
 		this.messages = new MessageSourceAccessor(messageSource);
 	}
 
+	/**
+	 * Sets the {@link SecurityContextHolderStrategy} to use. The default action is to use
+	 * the {@link SecurityContextHolderStrategy} stored in {@link SecurityContextHolder}.
+	 *
+	 * @since 6.0
```

Comment:
Good catch. Originally, this was targeted for 6.0. Then there was an attempt made to get it in for 5.7, which failed. Now that there is a 5.8 release, I've updated the `@since` tags to `5.8`